src/navigator/DrawerNavigator.tsx

```diff
@@ -313,11 +316,13 @@ export default function DrawerNavigator({ route }: Props) {
           initialParams={{ showDrawerTopBar: true }}
         />
       )}
-      <Drawer.Screen
-        name={Screens.FiatExchange}
-        component={FiatExchange}
-        options={{ title: t('addAndWithdraw'), drawerIcon: AddWithdraw }}
-      />
+      {showAddWithdrawOnMenu && (
```

Comment:
one side effect of this change is that the "Withdraw" h no longer works for the variant where this is hidden. I believe @jophish is working on making two different screens, one for the drawer and one for the home action, which we should also do for the Swap screen. Doesn't need to be on this PR and could be part of ACT-704, but maybe we wait on merging this until those changes go in since it breaks a variant?